### PR TITLE
 feat(client): delete `none` and `warning` from clientLogLevel

### PR DESCRIPTION
**Pull request by hiroppy**
ISSUE: https://github.com/webpack/webpack-dev-server/pull/1901

- [ ] This is a **bugfix**
- [x] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

yes

### Motivation / Use-Case

See the title.

### Breaking Changes

yes

### CODE REVIEW

**Comment by hiroppy** [comment]
Merge #1997 first.

File: client-src/default/utils/log.js [hunk 1]
```diff
@@ -26,17 +21,13 @@ export function setLogLevel(level) {
...
-      break;
-    // deprecated
-    case NONE:
     case SILENT:
       log.disableAll();
       break;
     default:
+      if (level === 'none' || level === 'warning') {
+        log.error('[WDS] "none" and "warning" have been deprecated.');
+      }
```
Comment (evilebottnawi) [inline client-src/default/utils/log.js]: Remove this, no need this in next release, it is major release

Comment (hiroppy) [reply client-src/default/utils/log.js]: fm, I want to avoid receiving many issues about these changes from everyone.😞

Comment (hiroppy) [reply client-src/default/utils/log.js]: ok, I'll delete here but I'll add warn to master. what do you think?

**Pull request merged**